python/note_input_widget/dialog.py

```diff
@@ -14,12 +14,27 @@
 class NoteInputDialog(QtGui.QDialog):
     """
     A dialog wrapper for the :class:`NoteInputWidget` widget.
+
+    The dialog instance will mirror all attributes and methods
+    found on the embedded :class:`NoteInputWidget` instance.
+    Example::
+
+        note_dialog = NoteInputDialog(parent=self)
+        note_dialog.entity_created.connect(self._on_entity_created)
+        note_dialog.data_updated.connect(self.rescan)
+        note_dialog.set_bg_task_manager(self._task_manager)
+        note_dialog.set_current_entity(self._entity_type, self._entity_id)
+
+        # show modal
+        note_dialog.exec_()
+
     """
-    def __init__(self, *args, **kwargs):
+    def __init__(self, parent):
```

Comment:
This is how we do it everywhere else in the code. I think in general, we should try to avoid the `*args, **kwargs` open syntax unless we have reason to use it. It can make it trickier to make changes in the future - say for example we wanted to add some parameters to this method in the future.